ai-gateway/app/routers/ingestion.py

```
@@ -0,0 +1,68 @@
+from pydantic import BaseModel
+import shutil
+import os
+import uuid
+
+from app.services.pdf_parser import parse_pdf
+from app.services.pptx_parser import parse_pptx
+from app.services.llm_generator import generate_summary_and_glossary
+
+router = APIRouter()
+
+TEMP_DIR = "/tmp/shiksha_ingestion"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/shiksha_ingestion"

(S108)